scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -43,10 +44,20 @@ def prep_and_run_tests(targeted_packages, python_version):
     parser.add_argument(
         'glob_string',
         nargs='?',
-        help = ('A comma separated list of glob strings that will target the top level directories that contain packages. '
+        help = ('A comma separated list of glob strings that will target the top level directories that contain packages.'
                 'Examples: All = "azure-*", Single = "azure-keyvault", Targeted Multiple = "azure-keyvault,azure-mgmt-resource"'))
 
+     #'--junitxml="junit/test-results.xml"'
```

Comment:
Clean up the default, give a bit more explanation as to the comment above. Like adding an `# example usage: ` on line 50.